index.html

```diff
@@ -4,12 +4,538 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
+    <title>Homepage</title>
 </head>
 
 <body>
-
-    
+    <header class="page-header">
+        <a href="#" class="main-logo"><img src="#" alt="pixmart logo">Pixmart.</a>
+        <nav class="page-nav">
+            <ul>
+                <li class="nav-item"><a href="#">About</a></li>
+                <li class="nav-item"><a href="#">Products</a></li>
+                <li class="nav-item"><a href="#">Creators</a>
+                    <p class="nav-item-tag">NEW</p>
+                </li>
+                <li class="nav-item"><a href="#">Blog</a></li>
+                <li class="nav-item"><a href="#">Contact</a></li>
+            </ul>
+            <div class="nav-buttons">
+                <button type="button" class="login-button">
+                    <img src="#" alt="login-logo"></button>
+                <button type="button" class="cart-button">
+                    <img src="#" alt="cart-logo"></button>
+            </div>
+        </nav>
+    </header>
+    <main class="page-main">
+        <section class="hero-section">
+            <!-- Question: Can I use figure to group the images without figcaption?

```

Comment:
yes, u can use figure without figcaption
because figure can work like a grouping tag for images